test/run_tests.cpp

```diff
@@ -1925,10 +1925,10 @@ static void run_test(TestCase *test_case) {
 
     if (test_case->is_parseh) {
         if (buf_len(&zig_stderr) > 0) {
-            printf("\nparseh emitted warnings:\n");
+            printf("\n!!!!! parseh emitted warnings:\n");
```

Comment:
Correct. I ended up disabling warning checking but only for that one test. Once we solve adding raw unions to the language (see #144) then we can correctly import these extra symbols.